Integrations/integration-qualys.yml

```diff
@@ -0,0 +1,2538 @@
+       'groupList': ['ids','id_min','id_max','truncation_limit','network_i1ds',
+                    'unit_id','user_id','title','show_attributes'],
+        'reportTemplateList': []
+
+    };
+
+
+
+    var methodHttpMethod = {
+        'reportList': 'GET',
+        'reportCancel': 'POST',
+        'reportDelete': 'POST',
+        'scorecardLaunch': 'POST',
+        'pcScanLaunch': 'POST',
+        'pcScanList': 'POST',
+        'reportFetch': 'POST',
+        'vmScanList': 'GET',
+        'vmScanAction': 'POST',
+        'vmScanLaunch': 'POST',
+        'vmScanFetch': 'GET',
+        'scapScanList': 'GET',
+        'pcScanManage': 'POST',
+        'scheduleScanList': 'GET',
+        'ipRestrictedList': 'GET',
+        'ipList': 'GET',
+        'ipAdd': 'POST',
+        'ipUpdate': 'POST',
+        'hostList': 'POST',
+        'virtualHostList': 'GET',
+        'virtualHostManage': 'POST',
+        'hostExcludedList': 'GET',
+        'hostExcludedManage': 'POST',
+        'scheduledReportList': 'GET',
+        'scheduledReportLaunch': 'POST',
+        'reportLaunchMap': 'POST',
+        'reportLaunchScanBasedFindings': 'POST',
+        'reportLaunchHostBasedFindings': 'POST',
+        'reportLaunchPatch': 'POST',
+        'reportLaunchRemediation': 'POST',
+        'reportLaunchCompliance': 'POST',
+        'reportLaunchCompliancePolicy': 'POST',
+        'vulnerabilityList': 'POST',
+        'groupList': 'GET',
+        'reportTemplateList': 'GET'
+    };
+
+
+
+    var titleDict = {
+        'reportList': 'Qualys Report List',
+        'reportCancel': 'Qualys Report Cancel',
+        'reportDelete': 'Qualys Report Delete',
+        'scorecardLaunch':'',
+        'pcScanList': '',
+        'reportFetch': '',
+        'vmScanList': 'Qualys VM Scan List',
+        'vmScanLaunch': '',
+        'vmScanAction': '',
+        'vmScanFetch': 'Qualys VM Scan',
+        'scapScanList': 'Qualys SCAP Scan List',
+        'pcScanManage': '',
+        'pcScanLaunch': '',
+        'scheduleScanList': '',
+        'ipRestrictedList': '',
+        'ipList': 'Qualys IP List',
+        'ipAdd': 'Qualys IP Add',
+        'ipUpdate': '',
+        'hostList': '',
+        'virtualHostList': '',
+        'virtualHostManage': '',
+        'hostExcludedList': 'Qualys Excluded Hosts List',
+        'hostExcludedManage': '',
+        'scheduledReportList': '',
+        'scheduledReportLaunch': '',
+        'reportLaunchMap': '',
+        'reportLaunchScanBasedFindings': '',
+        'reportLaunchHostBasedFindings':'',
+        'reportLaunchPatch':'',
+        'reportLaunchRemediation':'',
+        'reportLaunchCompliance':'',
+        'reportLaunchCompliancePolicy':'',
+        'vulnerabilityList': '',
+        'hostAssetList': '',
+        'groupList': 'Qualys Asset Groups List',
+        'reportTemplateList': 'Qualys Report Template List'
+    };
+
+
+    var entityDict = {
+        'report': [
```

Comment:
in the future please use one dictionart with a complex object (method, path, translator...)